src/main/java/io/turntabl/employementprofilingsystem/Controllers/EmployeeController.java

```diff
@@ -93,20 +101,29 @@
                         response.put("code","00");
                         response.put("msg","New employee added successfully");
                         response.put("data",key.longValue());
+                        childSpan.setTag("get_post_employee_status", "New Employee Creation Succeeded :)");
+
                     }else {
                         response.put("code","01");
                         response.put("msg","Failed to add new employee, try again later");
+                        childSpan.setTag("get_post_employee_status", "New Employee Creation Failed :(");
                     }
+
                 }
             }else {
                 response.put("code",result.get("code"));
                 response.put("msg",result.get("msg"));
+                childSpan.log("non-00 response code");
             }
         }catch (Exception e){
             e.printStackTrace();
             response.put("code","02");
             response.put("msg","Something went wrong, try again later");
+            span.log("Error has occurred");
```

Comment:
so what's a log and what's a span? And shouldn't the messages be the same? (Honestly, I don't know - should they?)